### DISCUSSION TITLE
[feat] Support ERC998

### DISCUSSION THREAD

**Discussion by TrejGun** [ProjectOpenSea/seaport#551] [Ideas]
I was in a team that developed [ERC998](http://erc998.org/) standard back in 2018 so I pretend to be a contributor and heavy user of this standard. Yes, it is not so popular because of reasons. One of these reasons marketplaces does not support it. You potentially could become the first one to support it with the new protocol. 

Basically, ERC998 utilizes the 4th optional parameter of `transferFrom` method from ERC721. It would be great if you can add it to `OfferItem` and `ConsiderationItem` and default to `0x`

In case you are interested let's convert this feature request into discussion and check the pros and cons

**Comment by Slokh** [comment]
I think we likely won't be adding this to v1.1, however definitely worth keeping in mind as a feature for future versions
